aggregate/housing_security/evictions_by_city_marshals.py

```diff
@@ -22,6 +30,11 @@ def load_residential_evictions(debug) -> pd.DataFrame:
         residential_evictions["borough"].str[0]
         + residential_evictions["borough"].str[1:].str.lower()
     )
+
+    # Hot fix, can be made neater
+    residential_evictions["borough_name"] = residential_evictions[
+        "borough_name"
+    ].replace({"Staten island": "Staten Island"})
```

Comment:
Got it.  This is the not elegant solution.  One idea for future improvement is to cast the value to all upper or lower case and then do a value match
i.e. set Staten Island to be "STATEN ISLAND" then map that value.